packages/vite-node/src/server.ts

```diff
@@ -98,7 +98,7 @@ export class ViteNodeServer {
   }
 
   shouldExternalize(id: string) {
-    return shouldExternalize(id, this.options.deps, this.externalizeCache)
```

Comment:
1. This code is processed by Vite, we need to check against actual code
2. If it's processed, then it was inlined

Reply:
Assuming that fetchCache already has a record, this means that this ·id· should be externalized, am I understanding this correctly? You can see that  https://github.com/vitest-dev/vitest/pull/4368/commits/20ed6ce9894515b65c07a8853ed56120f0e26f91

Reply:
If it's processed then it was already inlined yes, and this function will never even hit your new check because it's already cached. We don't need this new check at all I think.

Reply:
Yes, I thought wrong ...